test/UnitTests/PlatformServices.Shared.Unit.Tests/netstandard1.0/ns10DiaSessionOperationsTests.cs

```diff
@@ -0,0 +1,274 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace MSTestAdapter.PlatformServices.Tests.Services
+{
+#if NETCOREAPP1_0
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+#else
+    extern alias FrameworkV1;
+
+    using Assert = FrameworkV1::Microsoft.VisualStudio.TestTools.UnitTesting.Assert;
+    using TestClass = FrameworkV1::Microsoft.VisualStudio.TestTools.UnitTesting.TestClassAttribute;
+    using TestInitialize = FrameworkV1::Microsoft.VisualStudio.TestTools.UnitTesting.TestInitializeAttribute;
+    using TestMethod = FrameworkV1::Microsoft.VisualStudio.TestTools.UnitTesting.TestMethodAttribute;
+#endif
+
+    using System;
+    using System.IO;
+    using System.Reflection;
+    using Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices;
+    using MSTestAdapter.TestUtilities;
+#pragma warning disable SA1649 // SA1649FileNameMustMatchTypeName
+
+    [TestClass]
+    public class DiaSessionOperationsTests
```

Comment:
Thanks for getting these enabled. Can we close this issue now: https://github.com/Microsoft/testfx/issues/261